### PR TITLE
Sort text report entries by inhibitor status and risk factor

### PR DESCRIPTION
**Pull request by drehak**
Currently, the plaintext output entries are written in the order they were produced in. This makes it easy for an inhibitor to get lost, when it (at least in my opinion) should be at the very top.
This PR adds a simple `importance` function, which acts as a sort key. After inhibitors, entries with a high risk factor appear next. It can easily be added to the JSON report too, though in that one it makes sense to retain order by timestamp.

### CODE REVIEW

File: leapp/utils/report.py [hunk 1]
```diff
@@ -42,10 +42,24 @@ def fetch_upgrade_report_messages(context_id):
     return messages
 
 
+IMPORTANCES = {
```
Comment (bocekm) [inline leapp/utils/report.py]: Call it SEVERITY_LEVELS. Importance does not go well with Severity.

Comment (drehak) [reply leapp/utils/report.py]: Alright.

**Comment by drehak** [comment]
e2e tests

**Comment by drehak** [comment]
e2e tests

**Comment by drehak** [comment]
e2e tests

**Comment by drehak** [comment]
e2e tests

**Comment by drehak** [comment]
e2e tests

**Comment by drehak** [comment]
e2e tests

**Comment by bocekm** [comment]
e2e tests

**Comment by bocekm** [comment]
Please rebase and resolve the conflicts.

**Comment by drehak** [comment]
@bocekm done

**Pull request merged**